static/defaults.config.php

```diff
@@ -520,7 +520,7 @@
 		'profiler' => false,
 
 		// pushpoll_frequency (Integer)
-		// Frequency of contact poll for subhub contact using the DFRM or OStatus network.
+		// Frequency of contact poll for subhub contact using the DFRN or OStatus network.
```

Comment:
This _seems_ like a reasonable change, but I'm not particularly confident

Reply:
Nope, it's good.